sdlib/d/gc/page.d

```diff
@@ -984,6 +1013,11 @@ private:
 					continue;
 				}
 
+				// Clear any dead slots if they might contain pointers
+				if (containsPointers && evicted != 0) {
+					zeroSlabNimble(evicted, ec.sizeClass, e, 0);
+				}
```

Comment:
Do we have tests that check that the destructor gets the right values from the object it is destructing? Because now that we are zeroing things, we need to check that the order of things is proper.

Reply:
zeroing the destructor is going to zero the destructor pointer/flag. So if you call this out of order, the destructor won't be called.